nuxeo-drive-jsf/src/main/java/org/nuxeo/drive/seam/NuxeoDriveActions.java

```diff
@@ -157,6 +159,10 @@ public boolean canEditDocument(DocumentModel doc) {
         return getFileSystemItem(doc) != null;
     }
 
+    public boolean canEditBlob(DocumentModel doc, String xPath) {
+        return canEditDocument(doc) && doc.getPropertyValue(xPath) instanceof Blob;
```

Comment:
It's probably more efficient to do `doc.getProperty(xPath) instanceof BlobProperty` (untested)

Reply:
Actually this may not be the case, sorry, if you want to check that the blob is not null